.github/workflows/aws-heroku-helper.sh

```diff
@@ -59,10 +59,18 @@ function build_database() {
 
   print_header "Cloning Hasura Engine";
   git clone https://github.com/hasura/graphql-engine-heroku;
+  cd graphql-engine-heroku;
+  git checkout e947f74cf51ec586670140fc1181394c3d2f3300;
```

Comment:
In response to @johnclary's question about pinning `test` to a certain version of `graphql-engine`, this line is added to make sure we're running [exactly this commit](https://github.com/hasura/graphql-engine-heroku/commit/e947f74cf51ec586670140fc1181394c3d2f3300) out of the hasura/graphql-engine-heroku repo. This point-in-time mandates version 2.7.0 of `graphql-engine`.